lib/matplotlib/axes/_axes.py

```diff
@@ -6402,9 +6416,28 @@ def contourf(self, *args, **kwargs):
         return contours
     contourf.__doc__ = mcontour.QuadContourSet._contour_doc
 
-    def clabel(self, CS, *args, **kwargs):
-        return CS.clabel(*args, **kwargs)
-    clabel.__doc__ = mcontour.ContourSet.clabel.__doc__
+    def clabel(self, CS, levels=None, **kwargs):
+        """
+        Label a contour plot.
+
+        Adds labels to line contours in given `.ContourSet`.
+
+        Parameters
+        ----------
+        CS : `~.ContourSet` instance
+            Line contours to label.
+
+        levels : sequence of float
```

Comment:
Not sure how to best handle this. The only difference to `ControurSet.clabel` is the explicit `CS` parameter.

I think this is the best solution as it's not duplicating:
~~~
        Parameters
        ----------
        CS : `~.ContourSet` instance
            Line contours to label.
        **kwargs
            All other parameters are documented in `~.ContourLabeler.clabel`.
~~~
Alternatively, you can duplicate `levels` exactly. But I wouldn't do two different descriptions.

Reply:
I would suggest to describe `levels` further, because users probably won't click towards `~.ContourLabeler.clabel`  to understand what `Axes.clabel` method does ...